payments/direct-debit/payments.md

```diff
@@ -0,0 +1,465 @@
+      title: Credit Account Payments
+    - url: /payments/credit-card
+      title: Credit Card Payments
+    - url: /payments/invoice
+      title: Invoice Payments
+    - url: /payments/direct-debit
+      title: Direct Debit
+    - url: /payments/direct-debit/payments
+      title: Direct Debit Payments
+    - url: /payments/mobile-pay
+      title: Mobile Pay Payments
+    - url: /payments/swish
+      title: Swish Payments
+    - url: /payments/vipps
+      title: Vipps Payments
+---
+
+{% include alert.html type="warning"
+                      icon="warning"
+                      header="Site under development"
+                      body="The Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
+
+# Direct Debit Payment Pages
+
+>PayEx offer Direct debit bank payments for the Baltics (Estonia, Latvia and Lithuania).
+
+{:.table .table-striped}
+| **Country** | **Supported banks** |
+|![Estonia](https://developer.payex.com/xwiki/wiki/developer/download/Main/ecommerce/payex-payment-instruments/direct-debit-payments/direct-debit-payment-pages/WebHome/Estonia.png) Estonia | Swedbank |
+| ![Latvia](https://developer.payex.com/xwiki/wiki/developer/download/Main/ecommerce/payex-payment-instruments/direct-debit-payments/direct-debit-payment-pages/WebHome/Latvia_.png) Latvia | Swedbank|
+| ![Lithuania](https://developer.payex.com/xwiki/wiki/developer/download/Main/ecommerce/payex-payment-instruments/direct-debit-payments/direct-debit-payment-pages/WebHome/Lithuania.png) Lithuania | Swedbank |
+
+## Introduction
+
+*   When the consumer/end-user starts the purchase process in your merchant/webshop site, you need to make a `POST` request towards PayEx with your Purchase information. You receive a Redirect URL in return. 
+*   You need to redirect the consumer/end-user's browser to the Redirect URL.
+*   A bank selection page will be presented to the consumer/end-user.
+*   The consumer/end-user will be redirect to the choosen bank's login page where she have to verify her identity to continue the payment process. 
+*   PayEx will redirect the consumer/end-user's browser to one of two specified URLs, depending on whether the payment session is followed through completely or cancelled beforehand. Please note that both a successful and rejected payment reach completion, in contrast to a cancelled payment.
+*   When you detect that the consumer/end-user reach your completeUrl, you need to do a GET request to receive the state of the transaction.
+
+## Screenshots
+
+Screenshots  will be available at a later date.
+
+## API Requests
+
+The API requests are displayed in the [purchase flow](#HPurchaseflow). The options you can choose from when creating a payment with key operation set to Value Purchase are listed below. The general REST based API model is described in the [technical reference](https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/technical-reference/).
```

Comment:
Header links aren't prefixed with `H` and camel cased in Jekyll. I believe `#purchase-flow` might be the correct ID of the header, but you need to test it.